pantheon/src/test/java/tech/pegasys/pantheon/RunnerTest.java

```diff
@@ -342,6 +347,13 @@ private JsonRpcConfiguration jsonRpcConfiguration() {
     return configuration;
   }
 
+  private GraphQLRpcConfiguration graphQLRpcConfiguration() {
+    final GraphQLRpcConfiguration configuration = GraphQLRpcConfiguration.createDefault();
+    configuration.setPort(0);
+    configuration.setEnabled(true);
+    return configuration;
```

Comment:
GraphQL isn't required for this test so we should disable it.

Reply:
Anything I can provide further help, pls let me know

Reply:
done